lib/v2/mrdx/daily.js

```diff
@@ -0,0 +1,79 @@
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { getElementChildrenInnerText } = require('./utils');
+const dayjs = require('dayjs');
+const utc = require('dayjs/plugin/utc');
+dayjs.extend(utc);
+
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const dateFormatted = dayjs().utcOffset(8).format('YYYYMMDD');
+
+    const baseUrl = `http://mrdx.cn/content/${dateFormatted}`;
+    const link = `${baseUrl}/Page01DK.htm`;
+
+    const indexPageResponse = await got({
+        url: link,
+        method: 'get',
+    });
+    const $ = cheerio.load(indexPageResponse.data);
+
+    // 当日所有文章Url在<a>的`daoxiang` attribute中
+    // 其站点使用**非标准**的HTML语法
+    // eg: `<a class="foo" daoxiang="foo-bar.htm">Lorem ipsum</a>`
+    const passagesList = $('.listdaohang > ul > li a')
+        .toArray()
+        .map((item) => ({
+            link: new URL(`${baseUrl}/${item.attribs.daoxiang}`).href,
+        }));
+
+    const passages = await Promise.all(
+        passagesList.map((passage) =>
+            ctx.cache.tryGet(passage.link, async () => {
+                const response = await got({
+                    url: passage.link,
+                    method: 'get',
+                });
+
+                const $ = cheerio.load(response.data);
+
+                // 标题在`h2`,`h3`,`h4`,`h5`中，有多余空格和换行符
+                // 但是Middleware会处理
+                const title =
+                    getElementChildrenInnerText($('.bggray > .margin15 > h2')[0]) +
+                    getElementChildrenInnerText($('.bggray > .margin15 > h3')[0]) +
+                    getElementChildrenInnerText($('.bggray > .margin15 > h4')[0]) +
+                    getElementChildrenInnerText($('.bggray > .margin15 > h5')[0]);
+
+                // 作者和发布时间
+                // 中间用超多空格分隔
+                // 头尾均有多余空格
+                // eg: 新华每日电讯     2022年12月21日
+                const [author, pubDate] = getElementChildrenInnerText($('.shijian')[0])
+                    .split(' ')
+                    .filter((item) => item !== '')
+                    .map((item) => item.trim());
+
+                const content = $('.contenttext').html();
+
+                return {
+                    title,
+                    author,
+                    pubDate: parseDate(pubDate, 'YYYY年MM月DD日'),
+                    guid: link,
+                    link,
```

Comment:
> guid must be unique. link (defined from line 14) is not unique.

https://github.com/DIYgod/RSSHub/pull/11487#discussion_r1055499984
```suggestion
                    link: passage.link,
```

Reply:
This issue needs to be addressed before merging.
As you saw in the bot's preview, every feed item shares the **same** guid.
<details>
<summary>Feed 1</summary>

![1](https://user-images.githubusercontent.com/11386903/209466592-d537fae0-9502-4012-a9d7-bccb621b3d3a.PNG)
</details>
<details>
<summary>Feed 2</summary>

![2](https://user-images.githubusercontent.com/11386903/209466595-9ecb128a-2004-4eca-bdbf-78c011e4b69c.PNG)
</details>
<details>
<summary>Feed 3</summary>

![3](https://user-images.githubusercontent.com/11386903/209466597-69f6104f-65fa-4a5e-8337-18ba525edaf4.PNG)
</details>

Reply:
My bad. Now fixed.